jobconfig.toml

```diff
@@ -259,8 +261,8 @@
 [SECURITY_OFFICER]
 "Playtime Requirements" = 300
 "Required Account Age" = 0
-"Spawn Positions" = 4
-"Total Positions" = 4
+"Spawn Positions" = 7
+"Total Positions" = 7
```

Comment:
Keep this as-is, this is highly sensitive to balance and should be discussed in a separate PR.

Reply:
Change reverted, will change in accordance to balance discussion alongside Detective.